packages/core/src/scxml.ts

```diff
@@ -104,24 +104,21 @@ const evaluateExecutableContent = <
   meta: SCXMLEventMeta<TEvent>,
   body: string
 ) => {
-  const datamodel = context
-    ? keys(context)
-        .map((key) => `const ${key} = context['${key}'];`)
-        .join('\n')
-    : '';
-
-  const scope = ['const _sessionid = "NOT_IMPLEMENTED";', datamodel]
+  const scope = ['const _sessionid = "NOT_IMPLEMENTED";']
     .filter(Boolean)
     .join('\n');
 
   const args = ['context', '_event'];
 
   const fnBody = `
-    ${scope}
-    ${body}
+${scope}
+with (context) {
```

Comment:
Asking for forgiveness for using the deprecated `with` statement, but ✨ _it works so well_ ✨